src/app/components/pokemons/pokemons.component.ts

```diff
@@ -64,4 +75,13 @@ export class PokemonsComponent implements OnInit {
       );
     }
   }
+
+  loadSearchHistory(): void {
+    if (typeof localStorage !== 'undefined') {
```

Comment:
why?

Reply:
there is an error if the local storage is not initialized yet